src/ape/managers/compilers.py

```diff
@@ -51,15 +47,15 @@ def registered_compilers(self) -> Dict[str, CompilerAPI]:
             to compiler API classes.
         """
 
-        cache_key = self.config.PROJECT_FOLDER
+        cache_key = self.config_manager.PROJECT_FOLDER
         if cache_key in self._registered_compilers_cache:
             return self._registered_compilers_cache[cache_key]
 
         registered_compilers = {}
 
         for plugin_name, (extensions, compiler_class) in self.plugin_manager.register_compiler:
-            config = self.config.get_config(plugin_name)
```

Comment:
Caveat here is that now we have to make sure to load the configs from the compiler plugins. I have done this in the `ape-solidity` and it works fine I think! But I just want to make sure that we know this now and that this is really what we want.

Reply:
Adding back and leaving a NOTE comment